source/scale/elasticsearch.rst

```diff
@@ -69,7 +69,7 @@ Frequently Asked Questions (FAQ)
 Do I need to use Elasticsearch?
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The Elasticsearch engine is designed for large Enterprise deployments wanting to run highly efficient database searches in a cluster environment. The default Mattermost database search starts to show performance degradation at around 2.5 million posts, depending on the specifications for the database server. If you expect your Mattermost server to have more than 2.5 million posts, we recommend using Elasticsearch for optimum search performance.
+The Elasticsearch engine is designed for large Enterprise deployments to run highly efficient database searches in a cluster environment. The default Mattermost database search starts to show performance degradation at around 2.5 million posts, depending on the specifications for the database server. If you expect your Mattermost server to have more than 2.5 million posts, we recommend using Elasticsearch for optimum search performance.
```

Comment:
```suggestion
The Elasticsearch engine is designed for large Enterprise deployments that need to be able to run highly efficient database searches in a cluster environment. The default Mattermost database search starts to show performance degradation at around 2.5 million posts, depending on the specifications for the database server. If you expect your Mattermost server to have more than 2.5 million posts, we recommend using Elasticsearch for optimum search performance.
```

Reply:
I understand your change and it makes sense. I feel that we should specify that it's for deployments with this specific need - not all deployments will need this. Let me know what you think?

Reply:
Yeah it for this deployment with this specific need

Reply:
Yeah it is for this deployment with this specific need